ListDetail/src/androidTest/java/com/microsoft/device/display/samples/listdetail/ListDetailsInDualScreenModeTest.kt

```diff
@@ -30,7 +30,7 @@ import org.junit.runner.RunWith
 @LargeTest
 class ListDetailsInDualScreenModeTest {
     @get:Rule
-    val activityRule = ActivityTestRule<MainActivity>(MainActivity::class.java)
+    val activityRule = ActivityTestRule<ListDetailsActivity>(ListDetailsActivity::class.java)
```

Comment:
Maybe we could take advantage of this refactor and instead `ActivityTestRule` that is already deprecated we start using `ActivityScenario` or `ActivityScenarioRule` --> https://developer.android.com/reference/androidx/test/rule/ActivityTestRule

Reply:
This is valid for all `ActivityTestRule` usage along this PR ;-)

Reply:
I added ActivityScenarioRule, but also I added the testOrchestrator library. Without testOrchestrator (https://developer.android.com/training/testing/junit-runner#using-android-test-orchestrator), the tests are failing, I don't know exactly why.

Reply:
Maybe is required when you want `ActivityScenarioRule` to behave exactly as `ActivityTestRule` forlaunching an initial Activity? (just guessing...)

Reply:
I tried to manually launch the activity with the ActivityScenarioRule, but this doesn't solve the problem.

Reply:
We agreed then that is totally fine to go with the testOrchestrator too ;-)